tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_journal_mode.py

```diff
@@ -65,6 +66,20 @@ def run(ceph_cluster, **kw):
             ",".join(mon_node_ips),
             extra_params=f",fs={fs_name}",
         )
+        # down all the osd nodes
+        mdss = ceph_cluster.get_ceph_objects("mds")
+        mds_list = []
+        for mds in mdss:
+            daemon_name = fs_util.deamon_op(mds, "mds", "stop")
+            mds_list.append((mds, daemon_name))
+        log.info("All the mds nodes are down")
```

Comment:
@julpark-rh  -- Can you also include "def check_deamon_status" to validate the expected daemon status after stopping ??

Reply:
added